contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,647 @@
+// -----------------------------------
+//   module.Uri Library
+//   Builds a JSON array which to represent Realm metadata
+//
+// MIT License
+// -----------------------------------
+
+%lang starknet
+
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.registers import get_label_location
+from starkware.cairo.common.uint256 import Uint256, uint256_unsigned_div_rem
+from starkware.cairo.common.math import unsigned_div_rem
+from starkware.cairo.common.math_cmp import is_not_zero
+from starkware.cairo.common.bool import TRUE, FALSE
+
+from contracts.settling_game.utils.game_structs import RealmData
+
+namespace Utils {
+    namespace Symbols {
+        const LeftBracket = 123;
+        const RightBracket = 125;
+        const InvertedCommas = 34;
```

Comment:
What's inverted commas supposed to be? Because ASCII symbol 34 is `"` - standard double quote.

Reply:
Inverted Commas are another name for double quotes. It was what was in the file already so I stuck to that reference

Reply:
Wow. Today I learnt 🤓